### DISCUSSION TITLE
无法在服务器中使用

### DISCUSSION THREAD

**Discussion by AmaYadori98** [IndustrialCraftMC/IndustrialCraft2#143] [【漏洞反馈】Bugs Report]
使用该mod最新版（2.9.36-ex119）在服务器中（Minecraft1.19.2，Forge 43.3.0）中，服务端运作正常，但玩家加入时（游戏与forge版本同上）时会出现如下报错（除了这个还有说生铜块配方不对之类的）
服务端与客户端只装此mod仍出现此问题，排除与其它mod冲突可能
![dd1ddd38c9a9df6b2af348184b9f7de6](https://github.com/IndustrialCraftMC/IndustrialCraft2/assets/156582237/202d7d2c-e1e5-495a-8f26-967e94aeb8bc)

**Comment by Ashikaga-JFM** [comment]
同问
